oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_list.json

```diff
@@ -0,0 +1,51 @@
+[
+    {
+        "label": "PkiEnrollmentList",
+        "major_versions": [
+            2,
+            3
+        ],
+        "req": {
+            "cmd": "pki_enrollment_list",
+            "other_fields": []
+        },
+        "reps": [
+            {
+                "status": "ok",
+                "other_fields": [
+                    {
+                        "name": "enrollments",
+                        "type": "List<Nested7>"
+                    }
+                ]
+            }
+        ],
+        "nested_types": [
+            {
+                "label": "Nested7",
```

Comment:
`NestedXXX` is a random name provided by the script when it doesn't know how to retrieve it.

Here it should be named `PkiEnrollementListItem` I guess